client/state/a8c-for-agencies/agency/reducer.ts

```diff
@@ -18,7 +18,6 @@ export const initialState = {
 export const hasFetched = ( state = initialState.isFetching, action: AnyAction ) => {
 	switch ( action.type ) {
 		case JETPACK_GET_AGENCIES_SUCCESS:
-		case JETPACK_GET_AGENCIES_ERROR:
```

Comment:
Removed this because we want the` hasFetched` to be true only if the API succeded.